examples/pythonDriver/parametricSurfaceMapping/run_parametric_mesh_problem.py

```diff
@@ -0,0 +1,57 @@
+
+import sys
+import numpy as np
+from mpi4py import MPI
+import pygeosx
+from pygeosx_tools import wrapper
+
+
```

Comment:
@rrsettgast - Historically, I've used the parametric mesh mapping in legacy GEOS to conform the mesh to features such as surface topography.  Most of this process can be done using pygeosx.  If we want to support this sort of behavior, we will need a method to trigger a re-calculation of the element shape functions.

Note: Adding this in as a quick hack, I found that the gravitational stress field for this example appeared to be calculated correctly.


![deformed_mesh](https://user-images.githubusercontent.com/4932756/93269155-4fbf9500-f763-11ea-8389-0443f5659165.png)

Reply:
If this is a common pattern we could always add this as another point at which you return into the environment.